lib/ddtrace/metrics.rb

```diff
@@ -122,6 +130,26 @@ def initialize(*args)
       end
     end
 
+    # Return instance specific tags.
+    def instance_tags
+      tags = []
+      tags << "#{Ext::Metrics::TAG_ENV}:#{env}" if env
+      tags
+    end
+
+    # Add instance tags to default metrics
+    def default_metric_options
```

Comment:
Why rewrite this method? Could you simply add `options[:tags].concat(instance_tags)` to the existing method?

Reply:
I was able to simplify this logic, thank you for the feedback.